<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Filex</class>
 <widget class="QWidget" name="Filex">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Mode:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Single Folder </string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Percentage Complete:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>0%</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Files left to complete:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>Files will appear here. </string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="styleSheet">
            <string>
             QPushButton {
               background-color: #6200EA; /* Vibrant Purple */
               color: white;
               font-size: 16px;
               border-radius: 12px; /* Increased roundness */
               padding: 10px;
               border: none; /* Remove the border */
             }
             QPushButton:hover {
               background-color: #3700B3; /* Darker purple when hovered */
               transform: translateY(-2px); /* Slight lift effect */
             }
             QPushButton:pressed {
               background-color: #230036; /* Darker on press */
             }
            </string>
           </property>
           <property name="text">
            <string>Player</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="styleSheet">
            <string>
             QPushButton {
               background-color: #6200EA; /* Vibrant Purple */
               color: white;
               font-size: 16px;
               border-radius: 12px; /* Increased roundness */
               padding: 10px;
               border: none; /* Remove the border */
             }
             QPushButton:hover {
               background-color: #3700B3; /* Darker purple when hovered */
               transform: translateY(-2px); /* Slight lift effect */
             }
             QPushButton:pressed {
               background-color: #230036; /* Darker on press */
             }
            </string>
           </property>
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Files</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
